scripts/download_data.sh

```diff
@@ -90,8 +97,8 @@ if [[ -n ${DO_PET} ]]
 then
     echo Downloading PET data
 
-    mkdir -p ${DATA_PATH}/PET/mMR
-    pushd ${DATA_PATH}/PET/mMR
+    mkdir -p ${DOWNLOAD_DIR}
```

Comment:
you could replace this with 
```suggestion
    mkdir -p "${DOWNLOAD_DIR}"
```
to be safe for spaces. or with a few less characters `"$DOWNLOAD_DIR"`.

or leave it for another day